Util/channel/src/main/java/io/deephaven/util/channel/SeekableChannelsProviderLoader.java

```diff
@@ -54,4 +54,23 @@ public SeekableChannelsProvider fromServiceLoader(@NotNull final URI uri,
         }
         throw new UnsupportedOperationException("No plugin found for uri: " + uri);
     }
+
```

Comment:
I think we can remove the `fromServiceLoader(uri, specialInstructions)` path now, especially since we have a breaking release coming up.

Reply:
Just because one part of a release is breaking does not mean that we should feel free to break anything; there should be some level of nuance involved involved in each breaking change.

I did a quick scan of DHE usages; they have one test usage of `fromServiceLoader` afaict.